ui/src/app/views/workspace-edit/component.html

```diff
@@ -71,7 +71,7 @@ <h4 class="section-header">Please select all data use categories that apply for
                   name="disease-focused-research"
                   id="disease-focused-research"
                   [(ngModel)]="workspace.researchPurpose.diseaseFocusedResearch"
-                  [clrDisabled]="mode == Mode.Edit">
+                  [clrDisabled]=!canEditDataSet>
```

Comment:
Nit: Can we change this to `canEditResearchPurpose` or something similar? This just reads to me like we are editing the actual CDR for the workspace.